GermanFilter/sections/specific.txt

```diff
@@ -99,6 +99,9 @@ drwindows.de,wiesbadenaktuell.de,rauchmeldungen.de,simfans.de,wn.de,cosmopolitan
 ! ##.id-Container--proBEEP
 hallo-muenchen.de,az-online.de,lokalo24.de,rosenheim24.de,kreiszeitung.de,nordbuzz.de,mannheim24.de,merkur.de,oktoberfest-live.de,tz.de##.id-Container--proBEEP
 !
+mbpassion.de##.adsbygoogle:upward(1)
```

Comment:
It looks like your rules is ok here, but this one has to be in another file. You can find hints at the header of each file.